packages/clay-core/src/side-panel/Header.tsx

```diff
@@ -56,6 +66,18 @@ export const Header = ({
 			)}
 		>
 			<div className="autofit-row">
+				{onBack ? (
+					<div className="autofit-col">
+						<button
+							aria-label={messages.backAriaLabel}
+							className="btn-outline-borderless btn-secondary ml-n2"
```

Comment:
@veroglez We can use the class `component-action` to get it to be the right size. We need to add `displayType={null}` to get rid of `btn-primary`. I really wish we didn't default to primary.

Reply:
Updated! Thanks a lot!

Reply:
On the other hand, I'm waiting for @drakonux confirmation regarding the margins for this task https://liferay.atlassian.net/browse/LPD-60257, so for now we'll leave it on hold.

Reply:
Finally, I’ve moved this task https://liferay.atlassian.net/browse/LPD-60257 to the [next story](https://liferay.atlassian.net/browse/LPD-47742), since adjusting the header paddings requires taking the side panel’s resizable behavior into account.